src/jobservice/logger/getter/file_getter.go

```diff
@@ -34,7 +34,7 @@ func (fg *FileGetter) Retrieve(logID string) ([]byte, error) {
 		return nil, errs.NoObjectFoundError(logID)
 	}
 
-	return os.ReadFile(fPath)
+	return tailLogFile(fPath, LogSizeLimit)
```

Comment:
Add limitation here means no way to retrieve job's entire logs from outside(whatever API or inside manager/controller)?

Reply:
Change the LogSizeLimit to be a configurable item in jobservice config file, if  0 or not configured, means no limitation on the log size returned.